### PR TITLE
Adds Research, Virology, Glass Maint Airlocks

### PR DESCRIPTION
**Pull request by 13spacemen**
I tried out SS14 after SS13, went to my favorite department science, and noticed the airlocks were ugly as hell
Then I realized SS14 was using the old "science airlocks" instead of TG's nice purple "research airlocks"
The green virology airlocks were also missing

So I added those airlocks and a glass maint airlock too

~~Only map I edited so far is atlas.yml, never coded for SS14 before; replaced science airlocks with research ones, and replaced the virology room medical airlocks with virology ones~~
airlockss14

Ideally all or most of the science airlocks should be replaced by the research airlocks which look a lot better

~~I can map them in on all maps but did only Atlas so far to test the waters~~

:cl:
- add: Added purple research airlocks, green virology airlocks, (not mapped in yet)

### CODE REVIEW

**Review by Emisse — CHANGES_REQUESTED** [review]
Remove the atlas.yml change otherwise it will mess up ongoing mapping PRs. Otherwise looks fine

**Comment by 13spacemen** [comment]
@Emisse done

**Review by Emisse — APPROVED** [review]

**Comment by Peptide90** [comment]
The green on virology is pretty muted compared to our colour schemes I feel.

The sprites are decent but make sure they’re not out of place. Also glass maint kind of defeats the purpose of maint being dark but it’s not a bad idea for some variety if used sparingly, it’ll have its place.

**Comment by EmoGarbage404** [comment]
All the sprites seem to have assembly states that don't match the art style.

**Comment by Emisse** [comment]
the assembly states are what all of our airlocks use already

**Comment by 13spacemen** [comment]
@Peptide90 these sprites are all from TG, the virology green airlocks are the old r4407 medical airlocks, they used to be green before TG made them blue
As for glass maint, it's just a variant

@EmoGarbage404 yeah I noticed that was the case for all the other airlocks, so I just used the same assembly state as them

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml [hunk 1]
```diff
@@ -159,6 +167,22 @@
...
+    access: [["Research"]]
+
+- type: entity
+  parent: AirlockResearch
+  id: AirlockMedicalResearchLocked
+  suffix: Medical/Research, Locked
+  components:
+  - type: AccessReader
+    access: [["Research"], ["Medical"]]
+
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml:185]: i think if we're going to add the darker purple research airlocks they should really just be sprite changes on the old sci airlocks, I don't see any reason to have both

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by metalgearsloth** [comment]
Do we have virology as a dept even?

**Comment by mirrorcult** [comment]
no but they're fine to have as flavor since we do have disease ward sections of med

**Comment by mirrorcult** [comment]
needs merging the research and sci airlocks as outlined then good to merge

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**